erica/infrastructure/sqlalchemy/alembic/README.md

```diff
@@ -0,0 +1,13 @@
+Generic single-database configuration.
+
+### create new revision
+
+```bash
+alembic revision -m"create erica request table"
+```
+
+### make database migrations
+
+```bash
+alembic upgrade head
```

Comment:
I think that both those commands need to be run in pipenv. I was not able to get it to run outside of pipenv.
```suggestion
pipenv run alembic revision -m"create erica request table"
```

### make database migrations

```bash
pipenv run alembic upgrade head
```

Reply:
yes, to have to run it within venv - so `pipenv shell` and afterwards the alembic stuff should be also possible